iceoryx_examples/icecubetray/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
```

Comment:
No hard feelings on the name. But I'll only change it if we come up with a cooler name :P

Reply:
Since we already have the attitude quality over quantity I would also extend this for quality over coolness.
I know it's hard! 
But if we confuse a new community member with cool misleading names no one is helped. We want to make it as easy as possible for everyone so that they can focus on their cool stuff as fast as possible!

Reply:
I agree, for sure we can be cool & provide quality at the same time.

I guess we need a vote on the name :smiley: 

* iceaccessrights
* icerights
* iceaccess
* icecubetray
* iceacl
* icecontrolaccess
* icecreamaccess
* controlaccesstoicecream

@budrus @elBoberido @marthtz @ithier Any other ideas? What do you think?

Reply:
how about ice_access_control? This is much more readable! (variation of the third from the bottom)

Reply:
@mossmaurice you didn't mark your suggestions with emojis so I had to choose the one from @elfenpiff which isn't ambiguous with any emoji ;)